yellowbrick/classifier/prcurve.py

```diff
@@ -366,7 +371,7 @@ def _get_y_scores(self, X):
 ## Quick Method
 ##########################################################################
 
-def precision_recall_curve(model, X, y, ax=None, train_size=0.8,
+def precision_recall_curve(model, X, y ,X_test=None,y_test=None, ax=None, train_size=0.8,
```

Comment:
minor typo: space before comma after `y` arg and no space after comma before `X_test` and `y_test`. Eventually, we will have a preformatter that will handle this stuff for us, we just haven't gotten to that point yet! (#456)

Reply:
corrected!